helm/prometheus-rules/Chart.yaml

```diff
@@ -4,7 +4,7 @@ engine: gotpl
 home: https://github.com/giantswarm/prometheus-rules
 icon: https://s.giantswarm.io/app-icons/1/png/default-app-light.png
 name: prometheus-rules
-appVersion: '[[ .AppVersion ]]'
+appVersion: '[[ .Version ]]'
```

Comment:
Is it an expected change?
Maybe it deserves a word in PR description and in changelog, as users may see an inconsistency in versions.

Reply:
No this came with merging conflicts with the master branch